atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -69,11 +69,11 @@ def run(self):
         build_json = get_build_json()
         current_platform = platform.processor() or 'x86_64'
         self.manifest_list_cache = {}
-        for nonce, parent in enumerate(sorted(self.workflow.builder.parent_images.keys())):
-            image = ImageName.parse(parent)
+        for nonce, parent in enumerate(sorted(self.workflow.builder.parent_images.keys(),
+                                              key=lambda parent_image: str(parent_image))):
```

Comment:
Can be simplified: `key=str`

Reply:
Fixed.

Reply:
I don't see this change pushed.

Reply:
Apparently you have to commit before you push.
Fixed for real this time.